db/migrate/20170111135212_rename_draft_to_published.rb

```diff
@@ -0,0 +1,5 @@
+class RenameDraftToPublished < ActiveRecord::Migration
+  def change
+    rename_column :events, :draft, :published
```

Comment:
Dont you need to invert the values too? Otherwise the unpublished (drafts) will become published, and the other way round